### ISSUE TITLE
ord() passing only string parameter

### ISSUE THREAD

**Issue by icetee** [mrclay/jsmin-php#10]
Since PHP 7 requires that `ord` function parameter passing only string type.

The original notice message:
```
PHP Deprecated: ord(): Passing null to parameter #1 ($character) of type string is deprecated
```

**Comment by eSilverStrike** [comment]
When's the next version planned to be released to support PHP 8.1?

**Comment by ctrlaltca** [comment]
As soon as projects are moving to adopt php 8.1, this is quite a showstopper for eg. Pradosoft/prado.
It would be nice to see this fixed, thank you.

**Comment by glensc** [comment]
@ctrlaltca did you think sending a pr perhaps rather than complaining?

**Comment by ctrlaltca** [comment]
Hey man, try to have a look at https://github.com/mrclay/jsmin-php/pull/13

**Comment by glensc** [comment]
Should be fixed with 2.4.1 release, closing
